This patch updates code and doc to move pine.conf from /usr/local/lib to /etc.
Some more fixes in the doc according the move /usr/spool/news -> /var/spool/news.
--- alpine-2.00.orig/pith/filter.c
+++ alpine-2.00/pith/filter.c
@@ -7640,7 +7640,7 @@
 		}
 		else if(!strcmp(s, "PINE_CONF_PATH")){
 #if defined(_WINDOWS) || !defined(SYSTEM_PINERC)
-		    p = "/usr/local/lib/pine.conf";
+		    p = "/etc/pine.conf";
 #else
 		    p = SYSTEM_PINERC;
 #endif
@@ -7649,7 +7649,7 @@
 #ifdef SYSTEM_PINERC_FIXED
 		    p = SYSTEM_PINERC_FIXED;
 #else
-		    p = "/usr/local/lib/pine.conf.fixed";
+		    p = "/etc/pine.conf.fixed";
 #endif
 		}
 		else if(!strcmp(s, "PINE_INFO_PATH")){
--- alpine-2.00.orig/doc/tech-notes.txt
+++ alpine-2.00/doc/tech-notes.txt
@@ -272,13 +272,13 @@
    The selection of which MTA to use depends on the settings of smtp-server,
    sendmail-path, and compile-time options. The first MTA specified in the
    following list is used:
-    1. _sendmail-path_ in /usr/local/lib/pine.conf.fixed
+    1. _sendmail-path_ in /etc/pine.conf.fixed
     2. _smtp-server_ in /usr/local/pine.conf.fixed
     3. _sendmail-path_ specified on the command line.
     4. _smtp-server_ specified on the command line.
     5. _sendmail-path_ in the user's .pinerc file.
     6. _smtp-server_ in the user's .pinerc file.
-    7. _sendmail-path_ in /usr/local/lib/pine.conf
+    7. _sendmail-path_ in /etc/pine.conf
     8. _smtp-server_ in /usr/local/pine.conf
     9. DF_SENDMAIL_PATH defined at compile time.
    10. SENDMAIL and SENDMAILFLAGS defined at compile time.
@@ -604,7 +604,7 @@
    the binaries manually, you may use make install to install them.
 
    There  are  three  optional auxiliary files: /usr/local/lib/pine.info,
-   /usr/local/lib/pine.conf, and /usr/local/lib/pine.conf.fixed. The file
+   /etc/pine.conf, and /etc/pine.conf.fixed. The file
    pine.info contains text on how to get further help on the local system. It
    is part of the help text for the main menu and should probably refer to the
    local help desk or the system administrator. If this file doesn't exist a
@@ -677,9 +677,9 @@
    This section lists the various files which _Alpine_ uses which are not email
    folders. All of these are the default names of files, they may vary based on
    _Alpine_'s configuration.
-   /usr/local/lib/pine.conf
+   /etc/pine.conf
           Pine's global configuration file.
-   /usr/local/lib/pine.conf.fixed
+   /etc/pine.conf.fixed
           Non-overridable global configuration file.
    /usr/local/lib/pine.info
           Local pointer to system administrator.
@@ -875,10 +875,10 @@
           Configuration: Prints a sample system configuration file to the
           screen  or  standard  output.  To  generate  an  initial system
           configuration file, execute
-                alpine -conf > /usr/local/lib/pine.conf
+                alpine -conf > /etc/pine.conf
           To generate a system configuration file using settings from an old
           system configuration file, execute
-                alpine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+                alpine -P old-pine.conf -conf > /etc/pine.conf
           A system configuration file is not required.
    -convert_sigs _-p pinerc_
           Convert  signatures  contained  in signature files into literal
@@ -976,7 +976,7 @@
           Pinerc may be either a local file or a remote configuration folder.
    -P _pinerc_
           Uses the named file as the system wide configuration file instead of
-          _/usr/local/lib/pine.conf_ on UNIX, or nothing on _PC-Alpine_. Pinerc
+          _/etc/pine.conf_ on UNIX, or nothing on _PC-Alpine_. Pinerc
           may be either a local file or a remote configuration folder.
    -passfile _passfile_
           This tells _Alpine_ what file should be used as the password file.
@@ -1183,8 +1183,8 @@
    configuration. In most cases, the compiled-in preferences will suit users
    and administrators just fine. When running _Alpine_ on a UNIX system, the
    default built-in configuration can be changed by setting variables in the
-   system     configuration     files,     /usr/local/lib/pine.conf    or
-   /usr/local/lib/pine.conf.fixed. (Actually, these files can be changed using
+   system     configuration     files,     /etc/pine.conf    or
+   /etc/pine.conf.fixed. (Actually, these files can be changed using
    the      configure     arguments     --with-system-pinerc=VALUE     or
    --with-system-fixed-pinerc=VALUE.) The location of the pine.conf file can be
    changed  with  the -P command line argument. Both _Alpine_ and _PC-Alpine_
@@ -2771,7 +2771,7 @@
    _news-spool-directory_
           This option tells _Alpine_ where to look for the "news spool" for
           newsgroups when accessing news locally, rather than via NNTP. The
-          default path is usually /usr/spool/news.
+          default path is usually /var/spool/news.
    _newsrc-path_
           This  option  overrides  the  default name _Alpine_ uses for your
           "newsrc" news status and subscription file. If set, _Alpine_ will
@@ -3679,7 +3679,7 @@
           For  Unix  _Alpine_ the program _ispell_ works well as an alternate
           spell  checker.  If  your Unix system has _ispell_ it is probably
           reasonable to make it the default speller by configuring it as the
-          default in the system configuration file, /usr/local/lib/pine.conf.
+          default in the system configuration file, /etc/pine.conf.
           If this option is not set, then the system's _spell_ command is used.
           The spell command does not work the same as the alternate speller. It
           produces a list of misspelled words on its standard output, instead,
@@ -4073,12 +4073,12 @@
           must exist as a full path or a path relative to your home directory).
           Now for an example:
 
-     url-viewers=_TEST("test -n '${DISPLAY}'")_ /usr/local/bin/netscape,
-     /usr/local/bin/lynx, C:\BIN\NETSCAPE.BAT
+     url-viewers=_TEST("test -n '${DISPLAY}'")_ /usr/bin/sensible-browser,
+     /usr/bin/lynx, C:\BIN\NETSCAPE.BAT
           This example shows that for the first browser in the list to be used
           the environment variable DISPLAY must be defined. If it is, then the
-          file /usr/local/bin/netscape must exist. If either condition is not
-          met, then the file /usr/local/bin/lynx must exist. If it doesn't,
+          file /usr/bin/sensible-browser must exist. If either condition is not
+          met, then the file /usr/bin/lynx must exist. If it doesn't,
           then the final path and file must exist. Note that the last entry is
           a DOS/Windows path. This is one way to support _Alpine_ running on
           more than one architecture with the same configuration file.
@@ -10471,9 +10471,9 @@
     4. a command line argument
     5. the system-wide _fixed_ configuration file (Unix _Alpine_ only)
 
-   The fixed configuration file is normally /usr/local/lib/pine.conf.fixed.
+   The fixed configuration file is normally /etc/pine.conf.fixed.
 
-   The system-wide configuration file is normally /usr/local/lib/pine.conf for
+   The system-wide configuration file is normally /etc/pine.conf for
    Unix  _Alpine_  and is normally not set for _PC-Alpine_. For _PC-Alpine_, if
    the  environment  variable  _$PINECONF_  is  set,  that  is used for the
    system-wide  configuration. This location can be set or changed on the
@@ -10722,7 +10722,7 @@
      * A program that implements the SMTP or ESMTP protocol via stdio.
      * An entry in /etc/services for the alternate service.
      * An entry in /etc/inetd.conf for the alternate service.
-     * An entry in /usr/local/lib/pine.conf, /usr/local/lib/pine.conf.fixed or
+     * An entry in /etc/pine.conf, /etc/pine.conf.fixed or
        ~/.pinerc.
      _________________________________________________________________
 
@@ -11688,7 +11688,7 @@
 
    The second selection is the standard UNIX print command. The default is
    _lpr_, but it can be changed on a system basis to anything so desired in
-   /usr/local/lib/pine.conf.
+   /etc/pine.conf.
 
    The third selection is the user's personal choice for a UNIX print command.
    The text to be printed is piped into the command. _Enscript_ or _lpr_ with
--- alpine-2.00.orig/doc/tech-notes/background.html
+++ alpine-2.00/doc/tech-notes/background.html
@@ -129,7 +129,7 @@
 <OL>
 
 <LI><EM>sendmail-path</EM> in
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>
+<CODE>/etc/pine.conf.fixed</CODE>
 
 <LI><EM>smtp-server</EM> in <CODE>/usr/local/pine.conf.fixed</CODE>
 
@@ -141,9 +141,9 @@
 
 <LI><EM>smtp-server</EM> in the user's <CODE>.pinerc</CODE> file. 
 
-<LI><EM>sendmail-path</EM> in <CODE>/usr/local/lib/pine.conf</CODE>
+<LI><EM>sendmail-path</EM> in <CODE>/etc/pine.conf</CODE>
 
-<LI><EM>smtp-server</EM> in <CODE>/usr/local/pine.conf</CODE>
+<LI><EM>smtp-server</EM> in <CODE>/etc/pine.conf</CODE>
 
 <LI><CODE>DF_SENDMAIL_PATH</CODE> defined at compile time.
 
--- alpine-2.00.orig/doc/tech-notes/config-notes.html
+++ alpine-2.00/doc/tech-notes/config-notes.html
@@ -931,10 +931,10 @@
 </OL>
 <P>
 The fixed configuration file is normally
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+<CODE>/etc/pine.conf.fixed</CODE>.
 <P>
 The system-wide configuration file is normally
-<CODE>/usr/local/lib/pine.conf</CODE> for Unix <EM>Alpine</EM> and is normally not
+<CODE>/etc/pine.conf</CODE> for Unix <EM>Alpine</EM> and is normally not
 set for <EM>PC-Alpine</EM>.
 For <EM>PC-Alpine</EM>, if the environment variable <EM>$PINECONF</EM> is set, that
 is used for the system-wide configuration.
@@ -1252,8 +1252,8 @@
 
 <LI> An entry in <CODE>/etc/inetd.conf</CODE> for the alternate service.
 
-<LI> An entry in <CODE>/usr/local/lib/pine.conf</CODE>,
-<CODE>/usr/local/lib/pine.conf.fixed</CODE> or <CODE>~/.pinerc</CODE>. 
+<LI> An entry in <CODE>/etc/pine.conf</CODE>,
+<CODE>/etc/pine.conf.fixed</CODE> or <CODE>~/.pinerc</CODE>. 
 
 </UL>
 
--- alpine-2.00.orig/doc/tech-notes/config.html
+++ alpine-2.00/doc/tech-notes/config.html
@@ -7,8 +7,8 @@
 configuration. In most cases, the compiled-in preferences will suit users
 and administrators just fine. When running <EM>Alpine</EM> on a UNIX system, the
 default built-in configuration can be changed by setting variables in the
-system configuration files, <CODE>/usr/local/lib/pine.conf</CODE>
-or <CODE>/usr/local/lib/pine.conf.fixed</CODE>.
+system configuration files, <CODE>/etc/pine.conf</CODE>
+or <CODE>/etc/pine.conf.fixed</CODE>.
 (Actually, these files can be changed using the configure arguments
 --with-system-pinerc=VALUE or --with-system-fixed-pinerc=VALUE.)
 The location of the pine.conf file can be changed with the -P command line
@@ -2326,7 +2326,7 @@
 
 <DD> This option tells <EM>Alpine</EM> where to look for the "news spool" for
 newsgroups when accessing news locally, rather than via NNTP.  The default
-path is usually <CODE>/usr/spool/news</CODE>. <P>
+path is usually <CODE>/var/spool/news</CODE>. <P>
 
 <DT> <A NAME="newsrc-path"><EM>newsrc-path</EM></A>
 
@@ -3686,7 +3686,7 @@
 alternate spell checker.
 If your Unix system has <EM>ispell</EM> it is probably reasonable to make
 it the default speller by configuring it as the default in the
-system configuration file, <CODE>/usr/local/lib/pine.conf</CODE>.
+system configuration file, <CODE>/etc/pine.conf</CODE>.
 
 <P>
 If this option is not set, then the system's <EM>spell</EM> command is used.
--- alpine-2.00.orig/doc/tech-notes/installation.html
+++ alpine-2.00/doc/tech-notes/installation.html
@@ -227,8 +227,8 @@
 
 There are three optional auxiliary files: 
 <CODE>/usr/local/lib/pine.info</CODE>,
-<CODE>/usr/local/lib/pine.conf</CODE>, and
-<CODE>/usr/local/lib/pine.conf.fixed</CODE>.  The file
+<CODE>/etc/pine.conf</CODE>, and
+<CODE>/etc/pine.conf.fixed</CODE>.  The file
 <CODE>pine.info</CODE> contains text on how to get further help on the
 local system.  It is part of the help text for the
 main menu and should probably refer to the local help desk or the system
@@ -318,11 +318,11 @@
 
 <DL COMPACT>
 
-<DT> /usr/local/lib/pine.conf
+<DT> /etc/pine.conf
 
 <DD> Pine's global configuration file. 
 
-<DT> /usr/local/lib/pine.conf.fixed
+<DT> /etc/pine.conf.fixed
 
 <DD> Non-overridable global configuration file. 
 
--- alpine-2.00.orig/doc/tech-notes/porting.html
+++ alpine-2.00/doc/tech-notes/porting.html
@@ -307,11 +307,11 @@
 <DT>
  ___
 <DD>
-Reading of <CODE>/usr/local/lib/pine.conf</CODE> 
+Reading of <CODE>/etc/pine.conf</CODE> 
 <DT>
  ___
 <DD>
-Fixing variables and features in <CODE>/usr/local/lib/pine.conf.fixed</CODE> 
+Fixing variables and features in <CODE>/etc/pine.conf.fixed</CODE> 
 <DT>
  ___
 <DD>
--- alpine-2.00.orig/doc/tech-notes/cmd-line.html
+++ alpine-2.00/doc/tech-notes/cmd-line.html
@@ -87,7 +87,7 @@
 file, execute
 
 <PRE><CODE>
-		alpine -conf > /usr/local/lib/pine.conf
+		alpine -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 
@@ -95,7 +95,7 @@
 system configuration file, execute
 
 <PRE><CODE>
-		alpine -P old-pine.conf -conf > /usr/local/lib/pine.conf
+		alpine -P old-pine.conf -conf > /etc/pine.conf
 </CODE></PRE>
 <P>
 A system configuration file is not required.
@@ -258,7 +258,7 @@
 <DT> -P <EM>pinerc</EM>
 
 <DD> Uses the named file as the system wide configuration file instead of
-<EM>/usr/local/lib/pine.conf</EM> on UNIX, or nothing on <EM>PC-Alpine</EM>.
+<EM>/etc/pine.conf</EM> on UNIX, or nothing on <EM>PC-Alpine</EM>.
 Pinerc may be either a local file or a remote configuration folder.
 <P>
 
--- alpine-2.00.orig/doc/tech-notes/low-level.html
+++ alpine-2.00/doc/tech-notes/low-level.html
@@ -787,7 +787,7 @@
 
 The second selection is the standard UNIX print command.  The default is
 <EM>lpr</EM>, but it can be changed on a system basis to anything so
-desired in <CODE>/usr/local/lib/pine.conf</CODE>. <P>
+desired in <CODE>/etc/pine.conf</CODE>. <P>
 
 The third selection is
 the user's personal choice for a UNIX print command.  The text to be
--- alpine-2.00.orig/doc/alpine.1	2008-08-22 13:40:16.000000000 -0700
+++ alpin-2.00/doc/alpine.1	2009-04-13 17:23:14.197375773 -0700
@@ -307,9 +307,9 @@
 .br
 /usr/local/lib/pine.info	Local pointer to system administrator.
 .br
-/usr/local/lib/pine.conf	System-wide configuration file.
+/etc/pine.conf	System-wide configuration file.
 .br
-/usr/local/lib/pine.conf.fixed	 Non-overridable configuration file.
+/etc/pine.conf.fixed	 Non-overridable configuration file.
 .br
 /tmp/.\\usr\\spool\\mail\\xxxx	Per-folder mailbox lock files.
 .br
--- alpine-2.00.orig/README	2008-03-14 16:53:39.674727886 -0700
+++ alpine-2.00/README	2009-04-13 17:42:54.961359479 -0700
@@ -107,7 +107,7 @@
         your system with no other files than this binary, and no
         modifications to your system. Optionally you may create two
         configuration files, /usr/local/lib/pine.conf and 
-        /usr/local/lib/pine.info. See the documentation for details.
+        /etc/pine.info. See the documentation for details.
  
  pico   The standalone editor similar to the Alpine message composer.
         This is a very simple straight forward text editor.
--- alpine-2.00.orig/doc/tech-notes/installation.html	2008-06-17 10:48:13.000000000 -0700
+++ alpine-2.00/doc/tech-notes/installation.html	2009-04-13 17:42:54.973342358 -0700
@@ -226,7 +226,7 @@
 them. <P>
 
 There are three optional auxiliary files: 
-<CODE>/usr/local/lib/pine.info</CODE>,
+<CODE>/etc/pine.info</CODE>,
 <CODE>/usr/local/lib/pine.conf</CODE>, and
 <CODE>/usr/local/lib/pine.conf.fixed</CODE>.  The file
 <CODE>pine.info</CODE> contains text on how to get further help on the
@@ -326,7 +326,7 @@
 
 <DD> Non-overridable global configuration file. 
 
-<DT> /usr/local/lib/pine.info
+<DT> /etc/pine.info
 
 <DD> Local pointer to system administrator. 
 
--- alpine-2.00.orig/doc/alpine.1	2008-08-22 13:40:16.000000000 -0700
+++ alpine-2.00/doc/alpine.1	2009-04-13 17:42:54.981343026 -0700
@@ -305,7 +305,7 @@
 .br
 /etc/mime.types	System-wide file ext. to MIME type mapping
 .br
-/usr/local/lib/pine.info	Local pointer to system administrator.
+/etc/pine.info	Local pointer to system administrator.
 .br
 /usr/local/lib/pine.conf	System-wide configuration file.
 .br
--- alpine-2.00.orig/doc/tech-notes.txt	2008-08-22 13:40:16.000000000 -0700
+++ alpine-2.00/doc/tech-notes.txt	2009-04-13 17:42:55.129339903 -0700
@@ -603,7 +603,7 @@
    into _Alpine_ so there are no _required_ auxiliary files. Instead of copying
    the binaries manually, you may use make install to install them.
 
-   There  are  three  optional auxiliary files: /usr/local/lib/pine.info,
+   There  are  three  optional auxiliary files: /etc/pine.info,
    /usr/local/lib/pine.conf, and /usr/local/lib/pine.conf.fixed. The file
    pine.info contains text on how to get further help on the local system. It
    is part of the help text for the main menu and should probably refer to the
@@ -681,7 +681,7 @@
           Pine's global configuration file.
    /usr/local/lib/pine.conf.fixed
           Non-overridable global configuration file.
-   /usr/local/lib/pine.info
+   /etc/pine.info
           Local pointer to system administrator.
    ~/.pinerc
           Personal configuration file for each user.
--- alpine-2.00.orig/pith/filter.c	2008-08-21 16:50:47.000000000 -0700
+++ alpine-2.00/pith/filter.c	2009-04-13 17:42:55.229348834 -0700
@@ -3134,7 +3134,7 @@
 #endif
 
 #ifndef SYSTEM_PINE_INFO_PATH
-#define SYSTEM_PINE_INFO_PATH "/usr/local/lib/pine.info"
+#define SYSTEM_PINE_INFO_PATH "/etc/pine.info"
 #endif
 #define CHTML_VAR_EXPAND(S) (!strcmp(S, "PINE_INFO_PATH")   \
 			     ? SYSTEM_PINE_INFO_PATH : S)
